en/appendices/5-3-migration-guide.rst

```diff
@@ -85,6 +85,11 @@ Cache
   with an array of server addresses to enable cluster mode.
 - Several :ref:`cache-events` were added to allow monitoring the caching behavior.
 
+Collection
+----------
+
+- ``Collectino::any()`` was added to replace ``Collection::some()`` with a more familiar name.
```

Comment:
The method should also be added to `core-libraries/collections.rst` and description of `some()` be updated to state it's an alias for `any()`.

Reply:
Sorry, didn't read your initial comment.